src/main/resources/db/migration/V1_0__initial_schema.sql

```diff
@@ -1,140 +1,115 @@
+create sequence if not exists contract_id_seq start with 1 increment by 1;
 
 create table attachment (
-    category enum ('CONTRACT','OTHER'),
-    extension varchar(255),
-    file varchar(255),
-    id bigint not null auto_increment,
-    mime_type varchar(255),
-    name varchar(255),
-    note varchar(255),
+    id                      bigint not null auto_increment,
+    contract_id             varchar(10) not null,
+    category                enum ('CONTRACT','OTHER'),
+    filename                varchar(255),
+    mime_type               varchar(255),
+    note                    varchar(255),
+    content                 longblob,
     primary key (id)
 ) engine=InnoDB;
 
 create table contract (
-    additional_terms varchar(255),
-    case_id bigint,
-    contract_id varchar(10),
-    description varchar(4096),
-    id bigint not null auto_increment,
-    index_terms varchar(255),
-    municipality_id varchar(255),
-    signed_by_witness bit,
-    status enum ('ACTIVE','DRAFT','TERMINATED'),
-    version integer,
+    id                      bigint not null auto_increment,
+    municipality_id         varchar(255),
+    contract_id             varchar(10),
+    version                 integer,
+    type                    enum ('LAND_LEASE'),
```

Comment:
Is enum a preferred datatype here?
Would it be a bad idea to have a String (varchar) here instead?

(in order to avoid flyway-scripts every time a new value is introduced)

(You could still have an enum in the Java code (i.e. entioty class), together with an `jakarta.persistence.AttributeConverter`)

Reply:
Will create a new task and update outside of this PR, since it's a rather big change